app/main/appium-method-handler.js

```diff
@@ -174,60 +174,84 @@ export default class AppiumMethodHandler {
     return await this._execute({methodName, args, skipRefresh});
   }
 
+  /**
+   * If the platformName has context switch command.
+   * Windows, macOS etc may have no an ability to calls contexts related commands.
+   * tvOS also has no ability to support WebView so far.
+   * So, for now, we want to avoid calling contexts related command to them.
+   *
+   * @param {?string} platformName Platform name to check if it support contexts command
+   * @returns {boolean} True if the platformName support context switch
+   *
+   */
+  _hasContextCommands (platformName) {
+    return (['android', 'ios'].includes(_.toLower(platformName)));
+  }
+
   async _getContextsSourceAndScreenshot () {
     let contexts, contextsError, currentContext, currentContextError, platformName,
         source, sourceError, screenshot, screenshotError, statBarHeight, windowSize, windowSizeError;
 
+    // Get current session capabilities is not in W3C spec,
+    // so potentially the request fails. Then, let's ignore the result.
     try {
-      currentContext = await this.driver.currentContext();
-    } catch (e) {
-      if (e.status === 6) {
-        throw e;
+      ({platformName, statBarHeight} = await this.driver.sessionCapabilities());
```

Comment:
@KazuCocoa 

You placed the command here now, it was lower and executed in the native context and will _break_ for Android. Let me check this for sure

Reply:
@KazuCocoa 

Yes, this fix breaks element inspection on Chrome browser. The position of the elements can't be determined so the can't be added in the screenshot.

**Before the fix:**
![image](https://user-images.githubusercontent.com/11979740/87034629-6c31d500-c1e8-11ea-9b1e-0207986884f0.png)

**After the fix:**
![image](https://user-images.githubusercontent.com/11979740/87034732-a13e2780-c1e8-11ea-9526-0a59ce754953.png)

The order in which the commands are now executed are not correct